packages/core/src/new-api/internal/new-execution/basic-execution-strategy.ts

```diff
@@ -90,7 +90,7 @@ export class BasicExecutionStrategy implements ExecutionStrategy {
       {
         id: 1,
         type: NetworkInteractionType.ONCHAIN_INTERACTION,
-        to: undefined,
+        to: executionState.contractAddress,
```

Comment:
This is the tweak to call transactions, `undefined` makes sense for a deploy but I think it should be `exState.contractAddress` for a general call.

Reply:
Yes, this was definitely a bug